scout/server/blueprints/panels/views.py

```diff
@@ -16,6 +17,17 @@
 panels_bp = Blueprint("panels", __name__, template_folder="templates")
 
 
+@panels_bp.route("/api/v1/panels/<panel_name>", methods=["GET", "POST"])
+@public_endpoint
+def api_panels(panel_name):
+    """Return JSON data about panels with a given panel name.
+    Returns all versions.
+    """
+    json_out = controllers.panels_to_json(store, panel_name)
```

Comment:
Why not just calling the store.gene_panels from here? I am a big fun of using controllers to avoid too much code in the views, but in this case you are replacing a line with another line..

Reply:
Same here, so I'd like to keep it like that. Compare the main view for panels, where there are now multiple calls to the store directly. It gets complex fast, so one extra context switch could be worth it.